### PR TITLE
275817 - i18n - manufacturer resource

### PR DESCRIPTION
**Pull request by MBausson**

### CODE REVIEW

**Review by B-Rass — APPROVED** [review]
LGTM @nicolas-brousse

File: app/views/manufacturers/index.html.erb
[the remark this answers is not in the document]
Comment (B-Rass) [reply app/views/manufacturers/index.html.erb:1]:
```rb
<%= render 'layouts/breadcrumb', breadcrumb_variables: {'Paramètres' => modeles_url, 'Fabricants' => ''} %>
```

File: app/views/manufacturers/_form.html.erb [hunk 1]
```diff
@@ -27,10 +31,10 @@
   <div class="form-group">
     <%= f.label :documentation_url, class: "col-sm-2 control-label" %>
     <div class="col-sm-10">
-      <%= f.text_field :documentation_url, placeholder: "https://url_du_frabricant.com/suite_de_l_url/%s/reste_s_il_y_a", class: "form-control" %>
+      <%= f.text_field :documentation_url, placeholder: t(".url_placeholder"), class: "form-control" %>
       <span class="text-muted">
         <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
-        <small>Information : il faut remplacer le numéro de série dans l'URL par <code>%s</code>.</small>
+        <small><%= t(".information").html_safe %></small>
```
[the remark this answers is not in the document]
Comment (nicolas-brousse) [reply app/views/manufacturers/_form.html.erb]: `raw` call `html_safe` https://github.com/rails/rails/blob/23938052acd773fa24068debe56cd892cbf8d868/actionview/lib/action_view/helpers/output_safety_helper.rb#L18.

The idea is more to rename locale key with `_html` suffix.

**Review by nicolas-brousse — CHANGES_REQUESTED** [review]
@MBausson there is conflicts to resolve by doing a rebase from `master`. Also I add a comment about the local with html, it need to be changed.

**Pull request merged**